.gitignore

```diff
@@ -0,0 +1,4 @@
+**/__pycache__/**
+
+# Core
+keentools/blender_independent_packages
```

Comment:
Core path is more specific
keentools/blender_independent_packages/pykeentools_loader/pykeentools